src/NetMQ/Core/Transports/Tcp/TcpConnector.cs

```diff
@@ -248,6 +248,13 @@ public void OutCompleted(SocketError socketError, int bytesTransferred)
                     // Set the TCP keep-alive option values to the underlying socket.
                     m_s.SetSocketOption(SocketOptionLevel.Socket, SocketOptionName.KeepAlive, m_options.TcpKeepalive);
 
+                  
+#if NET8_0_OR_GREATER
```

Comment:
Last comment in the same "general" idea, these options became available in net5/netcore3.1, might be worth to do a `#if NET` here?